micro_dl/networks/unet_stack_2D.py

```diff
@@ -16,11 +16,16 @@ def __init__(self, network_config, predict=False):
 
         :param dict network_config: dict with all network associated parameters
         """
+        if not predict:
+            # Depth is None when predicting
+            assert 'depth' in network_config and network_config['depth'] > 1,\
```

Comment:
Shouldn't depth be 1 for a 2D model?

Reply:
for stack_to_2D, depth corresponds to num_input_slices which has to be > 1 and num_output_slices=1. It is a hybrid network. Please let me know if that makes sense!